app/utils.py

```diff
@@ -161,3 +162,25 @@ def purge_nulls_from_data(data):
 def get_request_page_questions():
     json_payload = get_json_from_request()
     return json_payload.get('page_questions', [])
+
+
+def index_object(framework, doc_type, object_id, serialized_object):
+    try:
+        index_name = current_app.config['DM_FRAMEWORK_TO_ES_INDEX_MAPPING'][framework][doc_type]
+
+        try:
+            search_api_client.index(
+                index_name=index_name,
+                object_id=object_id,
+                serialized_object=serialized_object,
+                doc_type=doc_type,
+            )
+        except dmapiclient.HTTPError as e:
+            current_app.logger.warning(
+                'Failed to add {} object with id {} to {} index: {}'.format(
+                    doc_type, object_id, index_name, e.message))
+
+    except KeyError as e:
+        current_app.logger.warning(
```

Comment:
I reckon that should be an error, not a warning.

Reply:
`logger.error()` ?

Reply:
`logger.error` I think should be fine; I don't know how anyone else is searching Kibana, but I tend to do `loglevel: ERROR OR status:500` when doing the 2nd line thang?

We don't want to abandon the entire transaction just because indexing failed.